src/tess_asteroids/movingtpf.py

```diff
@@ -2945,7 +3073,7 @@ def _save_hdulist(
         # Create default file name
         if file_name is None:
             file_name = "tess-{0}-s{1:04}-{2}-{3}-shape{4}x{5}".format(
-                str(self.target).replace(" ", ""),
+                str(self.target).replace(" ", "").replace("/", ""),
```

Comment:
is this because of target names with special characters? I believe there's a built-in function to identify special character in a string.